### PR TITLE
Joey Gallotta Tutorial topic: Update a User's Email

### PR DESCRIPTION
**Pull request by gallottaj**

### CODE REVIEW

**Comment by gallottaj** [comment]
Please review this new tutorial topic.

File: docs/tutorials/update-user-email.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+---
+
+# Tutorial: Update a user's email
+
+In this tutorial, you'll learn the operations to call to update a user's email.
```
Comment (rbwatson) [inline docs/tutorials/update-user-email.md:7]:
```suggestion
In this tutorial, you'll learn how to update a user's email.
```
Would this work?

File: docs/tutorials/update-user-email.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Expect this tutorial to take about 15 minutes to complete.
+
+## Before you start
+
+Make sure you've completed the [Before you start a tutorial](before-you-start-a-tutorial) topic on the development system you'll use for the tutorial.
+
+## Update a user's email
+
+You can update an existing user's email in the To-Do service. To do so, `PUT` an [`email`](../api/user/id/email) resource containing the user's details.
```
Comment (rbwatson) [inline docs/tutorials/update-user-email.md:17]:
```suggestion
You can update an existing user's email in the To-Do service.
```
The 2nd sentence might be confusing as the procedure that follows looks like you need to do more than that.

File: docs/tutorials/update-user-email.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    ```js
+    {
+        "last_name": "Jones",
+        "first_name": "Jenny",
+        "email": "jen.jones@example.com",
+        "id": 5
+    }
+    ```
+
```
Comment (rbwatson) [inline docs/tutorials/update-user-email.md:55]:
```suggestion

## Next steps

```
To make it clear that the paragraph isn't part of the procedure.

**Review by rbwatson — CHANGES_REQUESTED** [review]
The tutorial looks pretty good.
A conclusion would make it clear that they've completed the task successfully.
The next steps are a nice touch.
I made some additional suggestions for clarity and brevity.

**Review by rbwatson — APPROVED** [review]
Rebased for merge.
Thanks for squashing!

**Pull request merged**